lib/policies/base_policy.js

```diff
@@ -53,22 +53,22 @@ class BasePolicy {
     /**
      * Total transaction timeout in milliseconds.
      *
-     * The <code>totalTimeout</code> is tracked on the client and sent to the
+     * The <code>timeout</code> is tracked on the client and sent to the
      * server along with the transaction in the wire protocol. The client will
      * most likely timeout first, but the server also has the capability to
      * timeout the transaction.
      *
-     * If <code>totalTimeout</code> is not zero and <code>totalTimeout</code>
+     * If <code>timeout</code> is not zero and <code>timeout</code>
      * is reached before the transaction completes, the transaction will return
      * error {@link module:aerospike/status.ERR_TIMEOUT|ERR_TIMEOUT}.
-     * If <code>totalTimeout</code> is zero, there will be no total time limit.
+     * If <code>timeout</code> is zero, there will be no total time limit.
      *
      * @type number
      * @default 1000
      */
     this.totalTimeout = props.totalTimeout
     if (this.totalTimeout === undefined) {
-      this.totalTimeout = props.timeout
+      this.totalTimeout = props.totalTimeout
```

Comment:
I think we can remove the fallback to use `timeout` if `totalTimeout` is not set. But as-is, this line is non-sensical since it's exactly the same as line 69 above.